From: mark_story <mark@mark-story.com>
Date: Thu, 18 Jun 2020 21:13:41 -0400
Subject: [PATCH] Fix SchemaShell configuration change.

Comparisons are done with === true. This fixes the set operation.

Fixes #14720
Origin: upstream, https://github.com/cakephp/cakephp/commit/931d3a8626f85610c9eaf27d2e153ceec38d0fea;
---
 system/xi3/lib/Cake/Console/Command/SchemaShell.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/system/xi3/lib/Cake/Console/Command/SchemaShell.php b/system/xi3/lib/Cake/Console/Command/SchemaShell.php
index f23c2b7..6a704bd 100644
--- a/system/xi3/lib/Cake/Console/Command/SchemaShell.php
+++ b/system/xi3/lib/Cake/Console/Command/SchemaShell.php
@@ -53,7 +53,7 @@ class SchemaShell extends AppShell {
 		$this->out('Cake Schema Shell');
 		$this->hr();
 
-		Configure::write('Cache.disable', 1);
+		Configure::write('Cache.disable', true);
 
 		$name = $path = $connection = $plugin = null;
 		if (!empty($this->params['name'])) {
